src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -55,12 +53,14 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
```

Comment:
It seems like clientConfig should just be initialized to defaultConfig() as soon as it's declared.